flutter-idea/src/io/flutter/view/FlutterView.java

```diff
@@ -275,7 +277,8 @@ private void addBrowserInspectorViewContent(FlutterApp app,
         color,
         UIUtil.getFontSize(UIUtil.FontSize.NORMAL),
         flutterSdkVersion,
-        WorkspaceCache.getInstance(app.getProject())
+        WorkspaceCache.getInstance(app.getProject()),
+        openOnAppLaunch ? DevToolsIdeFeature.ON_DEBUG_AUTOMATIC : DevToolsIdeFeature.TOOL_WINDOW
```

Comment:
is it possible that a user can have the openOnAppLaunch setting on, and also manually open the Inspector tool window?

Reply:
Hmm yeah a user could open their project, and then before they run their app open the inspector tool window. I can change how we're setting `openOnAppLaunch` to only be true if the tool window was automatically opened.